modin/pandas/base.py

```diff
@@ -3144,9 +3144,7 @@ def __getitem__(self, key):
         """
 
         def _get_new_resampler(key):
-            subset = self._dataframe[key]
-            resampler = type(self)(subset, *self.resample_args)
-            return resampler
+            return Resampler(self._dataframe[key], **self.resample_args)
```

Comment:
I think @YarShev was against my simplification - this indeed makes it harder to make a subclass if we create `Resampler` explicitly